### PR TITLE
Give radios and access groups, take duffelsbags

### LINKED ISSUE #32: add all command roles
- [ ] #33
- [ ] #37
- [ ] #41
- [ ] #45

### LINKED ISSUE #33: commanding officer
- [x] #34
- [ ] #35
- [x] #36
- [x] #50

### LINKED ISSUE #35: commanding officer loadout

### LINKED ISSUE #37: Executive officer
- [x] #38
- [ ] #39
- [x] #40
- [x] #51

### LINKED ISSUE #39: Executive officer loadout

### LINKED ISSUE #41: staff officer

### LINKED ISSUE #43: staff officer loadout

### LINKED ISSUE #45: senior enlisted advisor
- [x] #46
- [ ] #47
- [x] #48
- [x] #53

### LINKED ISSUE #47: senior enlisted advisor loadout

### LINKED ISSUE #49: add all auxiliary support roles
- [ ] #54
- [ ] #55
- [ ] #56
- [ ] #57

### LINKED ISSUE #54: Auxiliary Support Officer
- [x] #58
- [ ] #59
- [x] #60
- [x] #61

### LINKED ISSUE #55: Pilot Officer
- [x] #62
- [ ] #63
- [x] #64
- [x] #65

### LINKED ISSUE #56: Dropship Crew Chief
- [x] #66
- [ ] #67
- [x] #68
- [x] #69

### LINKED ISSUE #57: Intelligence Officer
- [x] #70
- [ ] #71
- [x] #72
- [x] #73

### LINKED ISSUE #59: Auxiliary Support Officer loadout

### LINKED ISSUE #63: Pilot Officer loadout

### LINKED ISSUE #67: Dropship Crew Chief loadout

### LINKED ISSUE #71: Intelligence Officer loadout

### LINKED ISSUE #74: add all mp roles
- [ ] #75
- [ ] #76
- [ ] #77

### LINKED ISSUE #75: Chief MP
- [x] #78
- [x] #79
- [ ] #80
- [x] #81

### LINKED ISSUE #76: Warden
- [x] #82
- [ ] #83
- [x] #84
- [x] #85

### LINKED ISSUE #77: Military Police
- [x] #86
- [ ] #87
- [x] #88
- [x] #89

### LINKED ISSUE #80: Chief MP loadout

### LINKED ISSUE #83: Warden loadout

### LINKED ISSUE #87: Military Police loadout

### LINKED ISSUE #91: add all marine roles
- [ ] #92
- [ ] #93
- [ ] #94
- [ ] #95
- [ ] #96
- [ ] #97
- [ ] #98

### LINKED ISSUE #92: Squad Leader
- [x] #99
- [ ] #100
- [x] #101
- [x] #102

### LINKED ISSUE #93: Fireteam Leader
- [x] #103
- [ ] #104
- [x] #105
- [x] #106

### LINKED ISSUE #94: Weapons Specialist
- [x] #107
- [ ] #108
- [x] #109
- [x] #110

### LINKED ISSUE #95: Smartgunner
- [x] #111
- [ ] #112
- [x] #113
- [x] #114

### LINKED ISSUE #96: Hospital Corpsman

### LINKED ISSUE #97: Combat Technician
- [x] #119
- [ ] #120
- [x] #121
- [x] #122

### LINKED ISSUE #98: Rifleman
- [x] #123
- [ ] #124
- [x] #125
- [x] #126

### LINKED ISSUE #100: Squad Leader loadout

### LINKED ISSUE #104: Fireteam Leader loadout

### LINKED ISSUE #108: Weapons Specialist loadout

### LINKED ISSUE #112: Smartgunner loadout

### LINKED ISSUE #116: Hospital Corpsman loadout

### LINKED ISSUE #120: Combat Technician loadout

### LINKED ISSUE #124: Rifleman loadout

### LINKED ISSUE #127: add all engineering roles
- [ ] #128
- [ ] #129
- [ ] #130

### LINKED ISSUE #128: Chief Engineer
- [x] #131
- [ ] #132
- [x] #133
- [x] #134

### LINKED ISSUE #129: Ordnance Tech
- [x] #137
- [ ] #138
- [x] #139
- [x] #140

### LINKED ISSUE #130: Maintenance Tech
- [x] #141
- [ ] #142
- [x] #143
- [x] #144

### LINKED ISSUE #132: Chief Engineer loadout

### LINKED ISSUE #138: Ordnance Tech loadout

### LINKED ISSUE #142: Maintenance Tech loadout

### LINKED ISSUE #145: add all medical/research roles
We might not end up adding the nurse unless they have some kind of importance, it seems more worthwhile to just have more doctor slots open. Wouldn't be a bad idea to have a intern role though, so maybe it's fine.

- [ ] #146
- [ ] #147
- [ ] #148
- [ ] #149

### LINKED ISSUE #146: CMO
- [x] #150
- [ ] #151
- [x] #152
- [x] #153

### LINKED ISSUE #147: Doctor
- [x] #154
- [ ] #155
- [x] #156
- [x] #157

### LINKED ISSUE #148: Nurse
- [x] #158
- [ ] #159
- [x] #160
- [x] #161

### LINKED ISSUE #149: Researcher
- [x] #162
- [ ] #163
- [x] #164
- [x] #165

### LINKED ISSUE #151: CMO loadout

### LINKED ISSUE #155: Doctor loadout

### LINKED ISSUE #159: Nurse loadout

### LINKED ISSUE #163: Researcher loadout

### LINKED ISSUE #166: add all supply roles
- [ ] #167
- [ ] #168
- [ ] #169

### LINKED ISSUE #167: Quartermaster
- [x] #170
- [ ] #171
- [x] #172
- [x] #173

### LINKED ISSUE #168: Cargo Tech
- [x] #174
- [ ] #175
- [x] #176
- [x] #177

### LINKED ISSUE #169: Mess Tech
- [x] #178
- [ ] #179
- [x] #180
- [x] #181

### LINKED ISSUE #171: Quartermaster loadout

### LINKED ISSUE #175: Cargo Tech loadout

### LINKED ISSUE #179: Mess Tech loadout

### LINKED ISSUE #184: Survivor
- [x] #187
- [ ] #188
- [x] #189
- [x] #190

### LINKED ISSUE #185: CM Combat Correspondent (Reporter)
- [x] #191
- [ ] #192
- [x] #193
- [x] #194

### LINKED ISSUE #188: Survivor loadout

### LINKED ISSUE #192: Combat Correspondent loadout

### PR DESCRIPTION
**Pull request by Tunguso4ka**
## About the PR
gives all roles cm headsets
takes from all roles duffelbags
adds colony headset and colonist id card.
adds theirs accessGroups to all roles.

- requires #612 
- partially handles #30 
- fix role issues:
    - fix #32 
        - fix  #33 
            - fix #35
        - fix #37 
            - fix #39
        - fix #41 
            - fix #43
        - fix #45
            - fix #47
    - fix #49
        - fix #54 
            - fix #59
        - fix #55
            - fix #63
        - fix #56
            - fix #67
        - fix #57 
            - fix #71
    - fix #74 
        - fix #75 
            - fix #80
        - fix #76 
            - fix #83
        - fix #77
            - fix #87
     - fix #91 
         - fix #92 
           - fix #100
         - fix #93 
           - fix #104
         - fix #94 
           - fix #108
         - fix #95 
           - fix #112
         - fix #96
           - fix #116
         - fix #97 
           - fix #120
         - fix #98
           - fix #124
  - fix #127 
      - fix #128 
        - fix #132
      - fix #129 
        - fix #138
      - fix #130  
        - fix #142
   - fix #145
     - fix #146 
       - fix #151
     - fix #147 
       - fix #155
     - fix #148 
       - fix #159
     - fix #149
       - fix #163
    - fix #166 
      - fix #167 
        - fix #171
      - fix #168 
        - fix #175
      - fix #169
        - fix #179
    - fix some non-uscm roles (Second reporter role was added)
      - fix #184 
        - fix #188
      - fix #185
        - fix #192
      

## Why / Balance
radios because they need to hear, duh
duffels cause there`s no duffels in cm13

### CODE REVIEW

**Comment by Tunguso4ka** [comment]
Ready to review.

**Review by QuietlyWhisper — COMMENTED** [review]
looks fine but maybe draft this and I'll get access finished first

**Comment by Tunguso4ka** [comment]
yoo, description looks cool. Thanks Whisper.

File: Resources/Prototypes/_CM14/Roles/Jobs/Civilians/survivor.yml [hunk 1]
```diff
@@ -20,17 +20,18 @@
   joinNotifyCrew: false
   supervisors: job-supervisors-nobody
   canBeAntag: false
+  accessGroups:
+  - Colony #for playtest 1-2 only
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Roles/Jobs/Civilians/survivor.yml:24]: I forgot to make civilian colonist access
```suggestion
  - Colonist
```

File: Resources/Prototypes/_CM14/Roles/Jobs/Requisitions/Quartermaster.yml [hunk 2]
```diff
@@ -13,14 +13,10 @@
...
-  canBeAntag: true
-  access:
-  - Command
-  - Quartermaster
-  - Cargo
-  # prep
-  # alpha bravo charlie delta
+  supervisors: job-supervisors-command
+  canBeAntag: false
+  #accessGroups:
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Roles/Jobs/Requisitions/Quartermaster.yml:18]:
```suggestion
  accessGroups:
```

**Comment by Tunguso4ka** [comment]
gonna prat webedit didnt break tabulations.

**Comment by Tunguso4ka** [comment]
"partially fixes assign access to roles #30" will close that issue btw.

**Comment by Tunguso4ka** [comment]
Ready to review.

**Review by QuietlyWhisper — APPROVED** [review]

**Review by QuietlyWhisper — APPROVED** [review]

**Pull request merged**